src/containers/BankApi/BankApiAccounts/index.tsx

```diff
@@ -18,25 +18,31 @@ const BankApiAccounts: FC = () => {
 
       <DocEndpoint endpoint="/accounts" method="GET" />
       <RequestResponseSnippet
-        code={`[
-  {
-    "id": "9eca00a5-d925-454c-a8d6-ecbb26ec2f76",
-    "created_date": "2020-07-08T02:14:59.307535Z",
-    "modified_date": "2020-07-08T02:14:59.307553Z",
-    "account_number": "4d2ec91f37bc553bc538e91195669b666e26b2ea3e4e31507e38102a758d4f86",
-    "trust": "75.21"
-  },
-  {
-    "id": "ae4d43b0-5c34-4e56-8266-0e3531268815",
-    "created_date": "2020-07-08T02:15:12.271834Z",
-    "modified_date": "2020-07-08T02:15:12.271852Z",
-    "account_number": "a29baa6ba36f6db707f8f8dacfa82d5e8a28fa616e8cc96cf6d7790f551d79f2",
-    "trust": "94.63"
-  }
-]`}
+        code={`{
+  "count": 2,
+  "next": "http://143.110.137.54/accounts?limit=50&offset=50",
```

Comment:
add params table for this api as well and the ideal response to add I believe should be the response of the following url.
http://143.110.137.54/accounts?limit=2

Reply:
Fixed it for accs and for blocks too, to make it consistent